### DISCUSSION TITLE
RAGFlow fail to parse file

### DISCUSSION THREAD

**Discussion by xl76888** [infiniflow/ragflow#4273] [Q&A]
开始于:
Fri, 27 Dec 2024 16:52:17 GMT
持续时间:
11.52 s
进度:
Task has been received.
Page(1~100000001): Start to parse.
Page(1~100000001): [ERROR]Internal server error while chunking: [Errno 2] No such file or directory: kefuqa.txt
[ERROR]handle_task got exception, please check log

**Comment by JinHai-CN** [comment]
1. RAGFlow version?
2. How do you upload the file?
3. How about the system status? minio, elasticsearch ..

**Reply by xl76888** [reply]
解析问题已解决，
新问题：Elapsed:
6056.8 ms
Error:
HTTPConnectionPool(host='ragflow-minio', port=9000): Max retries exceeded with url: /txtxtxtxt1?location= (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7efc1de4b1f0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Storage:
minio

**Reply by xl76888** [reply]
RAGFlow version 0.15
